<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VhallLiveTitle</class>
 <widget class="QWidget" name="VhallLiveTitle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>50</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VhallLiveTitle</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#VhallLiveTitle{
background-color: rgb(64, 66, 72);
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
background-color: rgb(64, 66, 72);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labClassName">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(236, 236, 236);
font: 16px &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>小学英语零基础英语入门英语音标英语词汇英语口语</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>11</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_classLogo">
          <property name="minimumSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/idention/img/idention/id.jpg);
border-top-left-radius:2px;
border-bottom-left-radius:2px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_userName">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>91</width>
            <height>18</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(153, 153, 153);
font: 12px &quot;微软雅黑&quot;;
border: 1px solid rgb(153, 153, 153);
border-top-right-radius:2px;
border-bottom-right-radius:2px;

</string>
          </property>
          <property name="text">
           <string>edu_f3d2bf01</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="PushButton" name="btnNetState">
        <property name="minimumSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/button/img/button/netState_excellent.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="groupWdg" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="CCancelCountDownBtn" name="btnRaiseHands">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>28</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{	
font: 14px &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	border:1px;
	border-radius:14px;
	background-color: rgb(255, 159, 91);
}

QPushButton:hover{
	background-color: rgb(200, 159, 91);
}

QPushButton:pressed{
	background-color: rgb(255, 159, 91);
}

QPushButton:disable{
	background-color: rgb(255, 159, 91);
}

</string>
           </property>
           <property name="text">
            <string>举手上麦</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CTimeButton" name="btnBeginClass">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>28</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 14px &quot;微软雅黑&quot;;	
	color: rgb(255, 255, 255);
	border:1px;
	border-radius:14px;
	background-color: rgb(82, 204, 144);
}

QPushButton:hover{
	background-color: rgb(52, 184, 119);
}

QPushButton:pressed
{
	background-color: rgb(82, 204, 144);
}

QPushButton:disable{
	background-color: rgb(99, 224, 162);
}

</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMore">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{	
border-image: url(:/button/img/button/title_more_Normal.png);
}

</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>9</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnMin">
             <property name="minimumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton#btnMin{	
	border-image: url(:/Login/img/Login/min_black_Normal.png);
}

QPushButton#btnMin:hover{		
	
	border-image: url(:/Login/img/Login/mic_black_Click.png);
}

QPushButton#btnMin:pressed{
	border-image: url(:/Login/img/Login/min_black_Normal.png);
}

QPushButton:disable{
	border-image: url(:/Login/img/Login/min_black_Normal.png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnNomal">
             <property name="minimumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton#btnNomal{	
	border-image: url(:/button/img/button/title_Max__Normal.png);
}

QPushButton#btnNomal:hover{		
	border-image: url(:/button/img/button/title_Max_clicked.png);
}

QPushButton:pressed{
	border-image: url(:/button/img/button/title_Max__Normal.png);
}

QPussButton:disable{
	border-image: url(:/button/img/button/title_Max__Normal.png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnClose">
             <property name="minimumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">
QPushButton#btnClose{	
	border-image: url(:/button/img/button/title_close_Normal.png);
}

QPushButton#btnClose:hover{		
	border-image: url(:/button/img/button/title_close_Click.png);
}

QPushButton:pressed{
	border-image: url(:/button/img/button/title_close_Normal.png);
}

QPussButton:disable{
	border-image: url(:/button/img/button/title_close_Normal.png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header location="global">push_button.h</header>
  </customwidget>
  <customwidget>
   <class>CTimeButton</class>
   <extends>QPushButton</extends>
   <header location="global">ctimebutton.h</header>
  </customwidget>
  <customwidget>
   <class>CCancelCountDownBtn</class>
   <extends>QPushButton</extends>
   <header>CTimeButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../VLive/VLive.qrc"/>
 </resources>
 <connections/>
</ui>
